src/main/resources/application.properties

```
@@ -50,3 +50,4 @@ com.ampnet.projectservice.max-per-user=1_000_000_000_000_00
 
```

Add grpc clinet wallet-service config:
```suggestion
grpc.client.wallet-service.address=static://localhost:8228
grpc.client.wallet-service.negotiationType=PLAINTEXT
```

See line 36.